openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -93,6 +93,24 @@ OpenAssessment.ResponseView.prototype = {
         var handlePrepareUpload = function(eventData) { view.prepareUpload(eventData.target.files, uploadType); };
         sel.find('input[type=file]').on('change', handlePrepareUpload);
 
+        var handleCustomButton = function(eventData) {
```

Comment:
Could you add additional information about the button's type and event kind(i.e. onDownloadButtonClick) ?

Reply:
Done